segmentstore/server/src/main/java/io/pravega/segmentstore/server/tables/TableExtensionConfig.java

```diff
@@ -46,6 +46,7 @@
     public static final Property<Integer> SYSTEM_CRITICAL_MAX_UNINDEXED_LENGTH = Property.named("systemcritical.unindexed.bytes.max", EntrySerializer.MAX_BATCH_SIZE * 8);
     public static final Property<Integer> MAX_COMPACTION_SIZE = Property.named("compaction.bytes.max", EntrySerializer.MAX_SERIALIZATION_LENGTH * 4);
     public static final Property<Integer> COMPACTION_FREQUENCY = Property.named("compaction.frequency.millis", 30000);
+    public static final Property<Boolean> IS_COMPACTION_ENABLED = Property.named("compaction.enabled", true);
```

Comment:
tiny suggestion: `COMPACTION_ENABLED` is better than `IS_COMPACTION_ENABLED`. boolean fields don't have to contain the `is` prefix, otherwise, the getter && setter generated by the IntelliJ will be stranger:
```
isIsXXXX()
getIsXXXX()
```

Reply:
Ok, changed field name.